village/src/apps/gamecenter/components/puzzlepanel/blocks/TimeBlock.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+class TimeBlock extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    getMin() {
+        return Math.floor(this.props.seconds / 60);
+    }
+
+    getSec() {
+        return ('00' + (this.props.seconds % 60)).slice( -2 );
```

Comment:
[imo](in my opinion)
```suggestion
        return String(this.props.seconds % 60).padStart(2, '0');
```
こっちのがおしゃれっぽそう
https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/String/padStart